### PR TITLE
Add option to add tag resources

### PR DESCRIPTION
**Pull request by smelchior**
Make it possible to tag the s3 bucket and cloudtrail.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -53,4 +53,9 @@ variable "trail_bucket_default_encryption" {
 variable "trail_bucket_default_encryption_key" {
   description = "Defines if the KMS encryption key, used if trail_bucket_default_encryption is set to aws:kms"
   default     = ""
-}
\ No newline at end of file
+}
+
+variable "tags" {
```
Comment (derBroBro) [inline variables.tf]: We should add the Type here.

File: variables.tf [hunk 2]
```diff
@@ -53,4 +53,9 @@ variable "trail_bucket_default_encryption" {
 variable "trail_bucket_default_encryption_key" {
   description = "Defines if the KMS encryption key, used if trail_bucket_default_encryption is set to aws:kms"
   default     = ""
-}
\ No newline at end of file
+}
+
```
Comment (derBroBro) [inline variables.tf]: Can you please add a var for the Key-Name? It is ok if it defaults to "Name", but we for example use lowercase keys which could configured then.

**Comment by derBroBro** [comment]
@smelchior thank you for your PR and sorry for the delay checking this.
There are two changes where is would be awesome if you can change these.

**Comment by smelchior** [comment]
I made the requested changes, but the "Name" tag is pretty standard for AWS in that upercase notation i think:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-resources (Note the second paragraph)

https://github.com/terraform-aws-modules/terraform-aws-security-group/blob/master/main.tf#L11
https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/main.tf#L20

If you have a better suggestion for the variable name, let me know ;)

**Comment by derBroBro** [comment]
Thank you for the fast update. If you change also the CF tag by var.name_tag_name I can directly merge the request. 😄

**Comment by smelchior** [comment]
oh, forgot about that one. Could you also update the version in the terraform registry then?

**Comment by derBroBro** [comment]
Version should be online

**Pull request merged**